Update androidtv_remote.markdown (#37482)
home-assistant/home-assistant.github.io: source/_integrations/androidtv_remote.markdown

- Some devices experience disconnects every 15 seconds. This is typically resolved by rebooting the Android TV device after the initial setup of the integration.
- If you are not able to connect to the Android TV device, or are asked to pair it again and again, try force-stopping the Android TV Remote Service and clearing its storage. On the Android TV device, go to **Settings** > **Apps** > **Show system apps**. Then, select **Android TV Remote Service** > **Storage** > **Clear storage**. You will have to pair again.
- Some onscreen keyboards enabled by TV manufacturers do not support concurrent virtual and onscreen keyboard use. This presents whenever a text field is selected, such as "search" where a constant **use the keyboard on your mobile device** will show, preventing you from opening the onscreen keyboard to type. This can be overcome by either disabling your 3rd party keyboard and using the default Gboard keyboard or by deselecting **Enable IME** in the **Configure** page of the integration.

> - nVidia Shield devices have to have enabled options to wake up by any controller button, not only one with the nVidia logo. Otherwise they can't be turn on by this integration (https://github.com/home-assistant/core/issues/136111).

Reviewer: ```suggestion
- If you can't turn on your Nvidia Shield device, make sure you enable the xx option in ..
```
I took a stab at rephrasing this a bit. let me know if it is still correct.

Reviewer 2: I don't remember how the options are named. I'd change this to: If you can't turn on your Nvidia Shield device, make sure you enable "mention setting and how to access it".
I'd also remove the link to the GitHub issue. It doesn't add any value.
Keep in mind nVidia isn't the correct way to write it. Use the one in https://en.m.wikipedia.org/wiki/Nvidia_Shield_TV

Reviewer: @szymonzdziabek, do you know what the option is called exactly?

Author: I checked this, made pictures of the menu, so generally yes, however my system is in Polish :P I found however the menu screenshots in the web, and the proper options are located under: _Settings_ -> _Remotes & accessories_. I'll try to change the language and send you the English name of the options.
![Shield Wakeup](https://github.com/user-attachments/assets/28e156b0-d217-41b7-9e16-8def8815b310)

Author: English version below :)
Section: _Simplified wake buttons_
Options to be set to off:
- _SHIELD 2019 Remote: Wake on power and Netflix buttons only_
- _Controllers: Wake on NVIDIA or logo buttons only_

Reviewer: Nice! Thank you! Could you rephrase the description to include these strings?

Author: I'd love to, but that's my first time of GitHub, so could you please guide me how can I change this pull request? I've edited the file again and created the new commit.

Reviewer: Looks like that couldn't stop you. Obviously, you got this :muscle: :rocket:

> - If you can't turn on your Nvidia Shield device, make sure you haven't enabled options **SHIELD 2019 Remote: Wake on power and Netflix buttons only** and **Controllers: Wake on NVIDIA or logo buttons only** under **Settings** > **Remotes & accessories** > **Simplified wake buttons** section.

Reviewer: ```suggestion
- If you can't turn on your Nvidia Shield device, go to **Settings** > **Remotes & accessories** > **Simplified wake buttons** and disable the following options: **SHIELD 2019 Remote: Wake on power and Netflix buttons only** and **Controllers: Wake on NVIDIA or logo buttons only**.
```
